### PR TITLE
Feature/kak/detail categories#97

### LINKED ISSUE #97: Replace activities with categories on place detail screen
Currently we list a place's **activities** beneath its image carousel:

image

These should be replaced with the place's **categories**:

image

---

As for activities, we display cycling on the second line. We could add hiking and water rec there, too. But we'd have to use icons only, as showing both icons and labels will end up very messy. If we do so, we should probably remove the "cycling" label, and just use its icon, for consistency and cleanliness.

### PR DESCRIPTION
**Pull request by flibbertigibbet**
## Overview

List activities as strings at the top of the place detail view, instead of activities.

Activities (currently just cycling) are to be shown as icons below.

## Demo

screenshot_1530626254

Closes #97.

### CODE REVIEW

File: app/src/main/java/org/gophillygo/app/data/models/Destination.java [hunk 1]
```diff
@@ -76,10 +79,9 @@ public Destination(int id, int placeID, String name, boolean accessible, String
         this.categories = categories;
     }
 
-    public void setDistance(float distance) {
+    public void setDistance(Context context, float distance) {
```
Comment (maurizi) [inline app/src/main/java/org/gophillygo/app/data/models/Destination.java]: Without having a `public void setDistance(float distance) { ... }` method defined I can't get the project to compile.

I think a good way to fix this is might be to defer loading the resource until needed by changing `getFormattedDistance` to take a `Context` parameter. There is an implicit `context` variable according to S.O. https://stackoverflow.com/questions/32374483/android-databinding-where-to-get-context

Comment (flibbertigibbet) [reply app/src/main/java/org/gophillygo/app/data/models/Destination.java]: Adding a separate context-free public setter to satisfy Gson/Room is all that's needed. The build error didn't show until after a cache invalidation.

Deferring to the getter would defeat the advantage of only calculating it once and storing it, and there's context already available when and where it gets set.

**Comment by flibbertigibbet** [comment]
Updated.

File: app/src/main/java/org/gophillygo/app/data/models/Destination.java (hunk 1 above)
Comment (maurizi) [reply app/src/main/java/org/gophillygo/app/data/models/Destination.java]: https://github.com/azavea/cac-tripplanner-android/pull/116/commits/d76f73f97d991d9d487af9aee9d3c12f0d2c3a5a allowed the app to build, but now destinations do not show distance:

screenshot_20180703-145918

Note if I use the up navigation to return to the home activity the formatted distance actually shows up for short while but then is hidden.

Comment (flibbertigibbet) [reply app/src/main/java/org/gophillygo/app/data/models/Destination.java]: Yeah, looked like it should work, but silently fails to set anything. I'll go with the modifications to format on read instead.

**Comment by flibbertigibbet** [comment]
Took another pass at string formatting.

**Pull request merged**